product_inventory_service/conf/application.conf

```diff
@@ -9,11 +9,18 @@ slick.dbs.default.db.driver="org.postgresql.Driver"
 play.evolutions.db.default.autoApply=true
 
 # Assign default db credentials
-slick.dbs.default.db.url="jdbc:postgresql://localhost:8080/products"
+slick.dbs.default.db.url="jdbc:postgresql://localhost:5432/products"
 slick.dbs.default.db.user = "postgres"
 slick.dbs.default.db.password = "postgres"
 
 # Try to assign db credentials through environment variables
 slick.dbs.default.db.url="jdbc:postgresql://"${?POSTGRES_URL_PRODUCTS}
 slick.dbs.default.db.user=${?POSTGRES_USER}
 slick.dbs.default.db.password=${?POSTGRES_PASSWORD}
+
+# Vault config
+vault.api.path="http://localhost:????/v1"
+vault.api.path=${?VAULT_API_PATH}
```

Comment:
- The local configuration's port should be the same as in the Makefile
- To point the service to Vault when it's containerized, you need to specify this env var somewhere, e.g. in docker-compose